test/Mono.Linker.Tests/TestCasesRunner/TestCaseSandbox.cs

```diff
@@ -30,8 +42,17 @@ public TestCaseSandbox (TestCase testCase, string rootTemporaryDirectory, string
 		public TestCaseSandbox (TestCase testCase, NPath rootTemporaryDirectory, string namePrefix)
 		{
 			_testCase = testCase;
-			var name = string.IsNullOrEmpty (namePrefix) ? "linker_tests" : $"{namePrefix}_linker_tests";
-			_directory = rootTemporaryDirectory.Combine (name);
+
+			_directory = rootTemporaryDirectory.Combine (string.IsNullOrEmpty (namePrefix) ? "linker_tests" : namePrefix);
+
+			const string tcases_name = "Mono.Linker.Tests.Cases";
```

Comment:
It might be nice to generalize a bit.  I think `testCase. OriginalTestCaseAssemblyPath.FileNameWithoutExtension` would give the same behavior and work with our extended test framework where we have a `Unity.Linker.Tests.Cases` assembly

Reply:
Don't have a way to test that, so please submit PR with adjustments